C/CryptoMiniSat/build_tarballs.jl

```diff
@@ -48,8 +64,17 @@ dependencies = [
     Dependency("boost_jll"; compat="=1.71.0"),
     Dependency("Zlib_jll"),
     Dependency("SQLite_jll"),
-    Dependency("MPICH_jll")
 ]
 
+platforms, platform_dependencies = MPI.augment_platforms(platforms)
+# Avoid platforms where the MPI implementation isn't supported
+# OpenMPI
+platforms = filter(p -> !(p["mpi"] == "openmpi" && arch(p) == "armv6l" && libc(p) == "glibc"), platforms)
+# MPItrampoline
+platforms = filter(p -> !(p["mpi"] == "mpitrampoline" && libc(p) == "musl"), platforms)
+platforms = filter(p -> !(p["mpi"] == "mpitrampoline" && Sys.isfreebsd(p)), platforms)
+append!(dependencies, platform_dependencies)
+
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; preferred_gcc_version=v"7")
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
+               augment_platform_block, julia_compat="1.6", preferred_gcc_version=v"7")
```

Comment:
Changing julia compat _**requires**_ changing version number: #2763.  And if you change version number, you can use a newer boost_jll.

Reply:
Are you recommending to use the Yggdrasil-current Boost (1.76.0) or a new Boost-current Boost (1.79.0)?

Reply:
Eh, when packages break ABI so frequently, I prefer to build new versions only when really necessary (e.g. for new features needed by other packages, etc.), otherwise the risk is to fracture the ecosystem with too many compat constraints, so I'd use 1.7.6 if possible.

Reply:
Will do.